iceoryx2-bb/container/src/semantic_string.rs

```diff
@@ -379,6 +379,48 @@ macro_rules! semantic_string {
             value: iceoryx2_bb_container::byte_string::FixedSizeByteString<$capacity>
         }
 
+        // START: serde
```

Comment:
Are you using this tag also for code folding? I know of some IDEs/editors which work with `BEGIN/END` but haven't seen `START/END`

Reply:
I do not use this in the IDE and in the past we had no one who used this feature. It is for me a direct code comment so that I can see easily where all the serde related code is.

Reply:
Would you be fine if someone, *cough* me *cough*, would change `START` to `BEGIN`?